src/plugins/data_view_editor/public/components/form_fields/title_docs_popover.tsx

```diff
@@ -0,0 +1,124 @@
+          defaultMessage: 'Index pattern',
+        })}
+      </EuiPopoverTitle>
+      <EuiPanel
+        className="eui-yScroll"
+        css={css`
+          max-height: 40vh;
+          max-width: 500px;
+        `}
+        color="transparent"
+        paddingSize="m"
+      >
+        <EuiText size="s" data-test-subj="indexPatternDocsPopoverContent">
+          <p>
+            <FormattedMessage
+              id="indexPatternEditor.titleDocsPopover.startTypingDescription"
+              defaultMessage="Start typing in the Index pattern field, and Kibana looks for the names of indices, data streams, and aliases that match your input."
+            />
+          </p>
+          <ul>
+            <li>
+              <p>
+                <FormattedMessage
+                  id="indexPatternEditor.titleDocsPopover.useWildcardDescription"
+                  defaultMessage="To match multiple sources, use a wildcard (*)."
+                />
+              </p>
+              <p>
+                <FormattedMessage
+                  id="indexPatternEditor.titleDocsPopover.useWildcardExample"
+                  defaultMessage="{pattern} matches {index1}, {index2}, and so on."
+                  values={{
+                    pattern: <EuiCode>filebeat-*</EuiCode>,
+                    index1: <EuiCode>filebeat-apache-a</EuiCode>,
+                    index2: <EuiCode>filebeat-apache-b</EuiCode>,
+                  }}
+                />
+              </p>
+            </li>
+            <li>
+              <p>
+                <FormattedMessage
+                  id="indexPatternEditor.titleDocsPopover.useCommasDescription"
+                  defaultMessage="To match multiple single sources, enter their names, separated by a comma (,). Do not include a space after the comma."
+                />
+              </p>
+              <p>
+                <FormattedMessage
+                  id="indexPatternEditor.titleDocsPopover.useCommasExample"
+                  defaultMessage="{pattern} matches two indices"
+                  values={{
+                    pattern: <EuiCode>filebeat-a,filebeat-b</EuiCode>,
+                  }}
+                />
+              </p>
+            </li>
+            <li>
+              <p>
+                <FormattedMessage
+                  id="indexPatternEditor.titleDocsPopover.useMinusDescription"
+                  defaultMessage="To exclude a source, use a minus sign (-)."
+                />
+              </p>
+              <p>
+                <FormattedMessage
+                  id="indexPatternEditor.titleDocsPopover.useMinusExample"
+                  defaultMessage="For example, {pattern}."
+                  values={{
+                    pattern: <EuiCode>-test3</EuiCode>,
+                  }}
+                />
+              </p>
+            </li>
+          </ul>
+        </EuiText>
+      </EuiPanel>
+    </EuiPopover>
```

Comment:
Hi @gchaps,

Could you please review this new popover?
I pulled the text from docs.

<img width="600" alt="Screenshot 2023-03-01 at 12 02 46" src="https://user-images.githubusercontent.com/1415710/222221865-53524a22-5785-49c9-936e-982990aeefee.png">

Reply:
I'm tempted to shorted the index pattern field description text now that there's a popover to just the first sentence.

Reply:
I agree with the hint text under the field description.  I'll work up some text.

Reply:
Found that back in v7.10 we had the following messaging: `An index pattern can match a single source, for example, filebeat-4-3-22, or multiple data sources, filebeat-*.` 

Could it be a better hint for the input?

<img width="700" alt="Screenshot 2023-03-02 at 13 07 35" src="https://user-images.githubusercontent.com/1415710/222424826-b09af4e4-fdd6-4872-94eb-af43019c3653.png">